atomix/cluster/src/main/java/io/atomix/raft/RaftException.java

```diff
@@ -95,34 +95,13 @@ public OperationFailure(final RaftError.Type type, final String message, final O
     }
   }
 
-  public static class CommandFailure extends OperationFailure {
-
-    public CommandFailure(final String message, final Object... args) {
-      super(RaftError.Type.COMMAND_FAILURE, message, args);
-    }
-  }
-
   public static class QueryFailure extends OperationFailure {
 
     public QueryFailure(final String message, final Object... args) {
       super(RaftError.Type.QUERY_FAILURE, message, args);
     }
   }
```

Comment:
I would assume we can also remove this. Where is this used?

Reply:
nice catch